bitnami/thanos/templates/alert-rule/store_gateway.yml

```diff
@@ -5,7 +5,7 @@ Generated from https://github.com/thanos-io/thanos/blob/main/examples/alerts/ale
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: {{ template "common.names.fullname" . }}
+  name: {{ template "common.names.fullname" . }}-store
```

Comment:
I think it would be better to use the full name as the identifier. In this case, you would use store-gateway instead of store.